Population/Population Datamart/Update Datamart Small Area Population Estimates.R

```diff
@@ -112,13 +115,21 @@ datamart_output <- function(start, end, pop_name, file, file_name, template){
     
   }
   
+  # Due to the format of the datamart files, we need to manually remove several
+  # commas from the first line of each csv file
+  # Use this loop below to remove these commas from each file
+  # As this resaves the csv file, this section can take a long time to run
+  
+  gsub_dir(dir = datamart_filepath, pattern = "1269632,,,,,,,", 
+           replacement = "1269632", recursive = FALSE, ext = "csv")
+  
```

Comment:
This doesn't actually appear to be any quicker than manually removing the comma's. It works well for the larger areas as the files are much smaller but as this function edits the csv and then resaves with the comma's removed, it takes a long time for the file to resave. I'll try and find a more efficient way of doing this but this way does work, just takes a while to run.